lib/mrss/cluster_config.rb

```diff
@@ -209,7 +209,12 @@ def determine_cluster_config
       @server_version = build_info['version']
       @enterprise = build_info['modules'] && build_info['modules'].include?('enterprise')
 
-      @server_parameters = client.use(:admin).command(getParameter: '*').first
+      @server_parameters = begin
+        client.use(:admin).command(getParameter: '*').first
+      rescue => e
+        STDERR.puts("Failed to obtain server parameters: #{e.message}")
```

Comment:
I suggest
```
STDERR.puts("WARNING: Failed to obtain server parameters: #{e.class}: #{e.message}")
```

Reply:
Sounds good to me, updated as suggested.